### PR TITLE
fix manifest importing extra css

### PR DESCRIPTION
**Pull request by spuigmitjadxc**

### CODE REVIEW

File: app/assets/config/decidim_challenges_manifest.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
 //= link decidim/challenges/add_challenge.js
 //= link decidim/challenges/admin/challenges_form.js
 //= link_directory ../stylesheets/decidim/challenges .css
```
Comment (Leusev) [inline app/assets/config/decidim_challenges_manifest.js:3]: Be careful, there's a white space between the word "challenges" and ".css"

File: app/assets/config/decidim_problems_manifest.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+//= link decidim/challenges/add_challenge.js
+//= link decidim/challenges/admin/challenges_form.js
+//= link_directory ../stylesheets/decidim/problems .css
```
Comment (Leusev) [inline app/assets/config/decidim_problems_manifest.js:3]: Same here, be careful, there's a white space between the word "challenges" and ".css"
Furthermore, are *challenge* .js related files are needed here ?

File: app/assets/config/decidim_solutions_manifest.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+//= link decidim/challenges/add_challenge.js
```
Comment (Leusev) [inline app/assets/config/decidim_solutions_manifest.js:1]: Are *challenge* .js related files are needed here ?

File: app/assets/config/decidim_sdgs_manifest.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+//= link decidim/challenges/add_challenge.js
```
Comment (Leusev) [inline app/assets/config/decidim_sdgs_manifest.js:1]: Are *challenge* .js related files are needed here ?

**Review by Leusev — CHANGES_REQUESTED** [review]

**Comment by tramuntanal** [comment]
Tests succeeded but could not publish coverage

**Pull request merged**